src/@next/IndexTable/IndexTableStyle.ts

```diff
@@ -0,0 +1,1745 @@
+  }
+
+  .Polaris-Button__Icon:last-child {
+    margin-right: var(--pc-button-disclosure-icon-offset);
+    margin-left: var(--p-space-1);
+  }
+
+  .Polaris-Button__Icon + *:not(.Polaris-Button__Icon) {
+    margin-left: var(--p-space-1);
+  }
+
+  .Polaris-Button--sizeSlim {
+    min-height: var(--pc-button-slim-min-height);
+    padding: var(--pc-button-slim-vertical-padding) var(--p-space-3);
+  }
+
+  [data-buttongroup-segmented='true'] .Polaris-Button,
+  [data-buttongroup-segmented='true'] .Polaris-Button::after {
+    border-radius: 0;
+  }
+
+  [data-buttongroup-segmented='true'] > :first-child .Polaris-Button,
+  [data-buttongroup-segmented='true'] > :first-child .Polaris-Button::after {
+    border-radius: 0;
+    border-top-left-radius: var(--p-border-radius-1);
+    border-bottom-left-radius: var(--p-border-radius-1);
+  }
+
+  [data-buttongroup-segmented='true'] > :last-child .Polaris-Button,
+  [data-buttongroup-segmented='true'] > :last-child .Polaris-Button::after {
+    border-radius: 0;
+    border-top-right-radius: var(--p-border-radius-1);
+    border-bottom-right-radius: var(--p-border-radius-1);
+  }
+
+  [data-buttongroup-segmented='true'] > :last-child:first-child .Polaris-Button,
+  [data-buttongroup-segmented='true']
+    > :last-child:first-child
+    .Polaris-Button::after {
+    border-radius: var(--p-border-radius-1);
+  }
+
+  [data-buttongroup-connected-top='true'] > :first-child .Polaris-Button,
+  [data-buttongroup-connected-top='true']
+    > :first-child
+    .Polaris-Button::after {
+    border-top-left-radius: 0;
+  }
+
+  [data-buttongroup-connected-top='true'] > :last-child .Polaris-Button,
+  [data-buttongroup-connected-top='true'] > :last-child .Polaris-Button::after {
+    border-top-right-radius: 0;
+  }
+
+  [data-buttongroup-full-width='true'] .Polaris-Button {
+    display: flex;
+    width: 100%;
+  }
+
+  .Polaris-Stack {
+    --pc-stack-spacing: var(--p-space-4);
+    display: flex;
+    flex-wrap: wrap;
+    align-items: stretch;
+    margin-top: calc(var(--pc-stack-spacing) * -1);
+    margin-left: calc(var(--pc-stack-spacing) * -1);
+  }
+
+  .Polaris-Stack > .Polaris-Stack__Item {
+    margin-top: var(--pc-stack-spacing);
+    margin-left: var(--pc-stack-spacing);
+    max-width: 100%;
+  }
+
+  .Polaris-Stack--noWrap {
+    flex-wrap: nowrap;
+  }
+
+  .Polaris-Stack--spacingNone {
+    --pc-stack-spacing: 0;
+  }
+
+  .Polaris-Stack--alignmentCenter {
+    align-items: center;
+  }
+
+  .Polaris-Stack__Item {
+    flex: 0 0 auto;
+    min-width: 0;
+  }
+
+  .Polaris-Inline {
+    --pc-inline-gap-xs: initial;
+    --pc-inline-gap-sm: initial;
+    --pc-inline-gap-md: initial;
+    --pc-inline-gap-lg: initial;
+    --pc-inline-gap-xl: initial;
+    gap: var(--pc-inline-gap-xs);
+    display: flex;
+    flex-wrap: var(--pc-inline-wrap);
+    align-items: var(--pc-inline-block-align);
+    justify-content: var(--pc-inline-align);
+  }
+
+  @media (min-width: 30.625em) {
```

Comment:
Do we need this? Also same for all media queries

Reply:
ohh thanks for spotting this! yes, we can remove 👌